### PR TITLE
Show issue title

### LINKED ISSUE #51: Show issue title, not just code

### PR DESCRIPTION
**Pull request by mikx98**
Closes #51

### CODE REVIEW

File: server/apiCallbacks.js [hunk 1]
```diff
@@ -24,8 +24,13 @@ export const getIssueList = async (req, res) => {
       res.status(500).json({})
     } else {
       const dataObj = JSON.parse(data)
+
       const total = dataObj.total
-      const newIssues = dataObj.issues.map(issue => issue.key)
+      const newIssues = dataObj.issues.map(issue => ({
```
Comment (Siegrift) [inline server/apiCallbacks.js]: :+1: for no **t**

File: src/worklogs/components/issuesInputSelector.js [hunk 2]
```diff
@@ -1,30 +1,67 @@
...
   (recentIssues, allIssues) => uniq(recentIssues.concat(allIssues))
 )
 
+const combinedIssuesSelector: InputSelector<
+  State,
+  any,
+  Issue[]
+> = createSelector(
+  [combinedIssueKeysSelector, allIssuesObjSelector],
+  (keys, allIssuesObj) => keys.map(key => allIssuesObj[key]).filter(Boolean)
```
Comment (Siegrift) [inline src/worklogs/components/issuesInputSelector.js]: I didn't know you can pass Boolean to lodash's functions :+1:

File: server/apiCallbacks.js (hunk 1 above)
Comment (mikx98) [reply server/apiCallbacks.js]: thought of you when changing :smile:

File: src/worklogs/components/issuesInputSelector.js (hunk 2 above)
Comment (mikx98) [reply src/worklogs/components/issuesInputSelector.js]: Boolean is just like any other function pointer, so why not :)

**Pull request merged**